internal/pkg/admission/policy-server-service.go

```diff
@@ -3,16 +3,37 @@ package admission
 import (
 	"context"
 	"fmt"
+	"os"
+	"strconv"
 
 	policiesv1alpha2 "github.com/kubewarden/kubewarden-controller/apis/policies/v1alpha2"
 
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
I guess we're using that to address the potential buffer overflow when parsing a string back into an Int.

I haven't looked at the code, but I would prefer to use something provided by the core Go library. The reason is: I fear we might ran into security issues if we do not update this dependency.

According to [that](https://github.com/kubernetes/kubernetes/issues/81121), we could simply use [`strconv.ParseInt`](https://pkg.go.dev/strconv#ParseInt)

Reply:
I see :+1: 

Kubernetes really lives inside of its own dimension :)